vocab/properties.json

```diff
@@ -3531,7 +3767,9 @@
     },
     "description": "Non-abridged instructions, comments, and information for using a particular product.",
     "label": "Full documentation",
+    "labelPlural": "Full documentations",
```

Comment:
"documentations" is not wrong, but using just "documentation" for the plural feels more natural